interp/src/interpreter/steppers/control_interpreter.rs

```diff
@@ -455,22 +573,42 @@ impl<'a> Interpreter for IfInterpreter<'a> {
             vec![]
         }
     }
+
+    fn get_mut_env(&mut self) -> MutStateView<'_, 'outer> {
+        if let Some(cond) = &mut self.cond {
+            cond.get_mut_env()
+        } else if let Some(branch) = &mut self.branch_interp {
+            branch.get_mut_env()
+        } else {
+            unreachable!("Invalid internal state for IfInterpreter")
```

Comment:
A little more explanation for why this is invalid is needed. Same with the unreachable below.

Reply:
The if interpreter could either be evaluating the condition or one of the branches (both are in options). It shouldn't be the case that neither is executing since it "idles" in the finished branch state

Reply:
Thanks! Can you change the error message to reflect this?